pkgs/tools/misc/ppfetch/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenvNoCC, lib, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "ppfetch";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "yael333";
+    repo = "ppfetch";
+    rev = "acf00175113db7e4bcd04f40596494360d232fdf";
+    sha256 = "sha256-v1l+0MNY8aruoljf9jAzgrpgFiKgqdWTdLv4o78a8kU=";
+  };
+
+  dontBuild = true;
```

If there isn't a build phase there is probably also no configure phase, so it could also be disabled:
```suggestion
  dontConfigure = true;
  dontBuild = true;
```